algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -239,6 +234,14 @@ def greedy_decode(self, logits, logit_paddings):
     padding = (fin_result == 0)
     return fin_result, padding
 
+  def sync_sd(self, params):
```

Comment:
can we rename this to something like sync_batch_norm or something that says it is syncing only BN?

Reply:
Actually, I'm syncing the entire state_dict in this function. Should I change it to only sync the BN stats?

Reply:
what else is present in state_dict apart from batch norm stats ?

Reply:
All the parameters!

Reply:
Sorry I'm out of the loop, but why do we need to sync anything here? And why don't we use the `torch.nn.SyncBatchNorm.convert_sync_batchnorm` wrapper? Also, is this function taking the mean of the parameters (they should be synced already, so this effectively would do nothing)?

Reply:
Because we have our own BatchNorm implementation that takes padding as input and tracks stats.
Yes, it does nothing for the other parameters.

Reply:
I see, maybe it's better to explicitly exclude the parameters to avoid confusion.